cglims/api.py

```diff
@@ -39,8 +39,12 @@ def pipeline(self):
 
         Returns (str): 'mip' or 'mwgs'
         """
-        if self.lims.udf.get('tissue_type') != 'tumour':
-            return 'mip' if self.apptag.is_human else 'mwgs'
+        if self.lims.get('tissue_type') != 'tumour':
```

Comment:
uhmm.. the LIMS sample itself doesn't have this information, right? Is it a UDF?